content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/enforce-2fa.md

```diff
@@ -7,25 +7,26 @@ description: >
 ---
 
 {{% pageinfo %}}
-As an [Organization Owner](/getting-started/glossary/#organization-owner), you can enforce two-factor-authentication (2FA) for all users within your organization.
+As an [Organization Owner](/getting-started/glossary/#organization-owner), you can enforce two-factor authentication (2FA) for all organization users who authenticate with their email and password.
 {{% /pageinfo %}}
 
-2FA enforcement affects organization users and pentest collaborators, including the following roles:
+## Important to Know
 
-{{% owner-member-team-member %}}
+Note the following about 2FA enforcement:
 
-{{< alert title="Note" color="primary" >}}
-If your organization uses [SAML-based single sign-on (SSO)](/getting-started/sign-in/#saml-sso), you bypass any 2FA enforcement. Your SAML provider may already require 2FA.
-{{< /alert >}}
+- 2FA enforcement only applies to **users who authenticate with their email and password**.
+- 2FA enforcement doesn't apply to single sign-on (SSO). This includes [SAML single sign-on (SSO)](/getting-started/sign-in/#saml-sso) and Google authentication.
+- If your organization has SAML SSO enforced, you can't enforce 2FA. Your SAML provider may already require 2FA.
+- 2FA enforcement affects users with the following roles:
+    {{% owner-member-team-member %}}
 
 ## Enforce 2FA
 
 1. Navigate to **Settings** > **Identity & Access**.
-1. Under **Two-Factor Authentication**, select **Enforce 2FA**.<br><br>
-  ![Enforce 2FA for your organization](/deepdive/Enforce2FA.png "Enforce 2FA for your organization")
-1. In the overlay that appears, select **Yes, enforce 2FA** to confirm.
+1. Under **Enforce Two-Factor Authentication (2FA)**, turn on the toggle.
+1. In the overlay that appears, confirm your action.
```

Comment:
You'll see "enabled" label next to Enforce 2FA 
![Screenshot 2023-04-18 at 12 12 43 PM](https://user-images.githubusercontent.com/92950285/232853479-58620306-d81f-484b-9b52-295ba9044f10.png)